### ISSUE TITLE
Make this repository private?

### ISSUE THREAD

**Issue by johnousterhout** [cpt-chewieeee/POS_project#1]
I'm a Professor at Stanford, and I'm teaching an Operating Systems course that uses Pintos for its class projects. I'm writing because an automated search of GitHub identified this repository as possibly containing solutions for the Pintos programming projects. This is a problem for us, because it makes it very easy for students to cheat. Even if they just look at your solution, and don't copy it, that will still reduce the educational experience for them. If this repo does indeed contain Pintos solutions, would you be willing to take it down or at least make it private? If so, that would be a great help to us. If for some reason you absolutely must keep the repository public, it would still be a help if you could rename it so that it doesn't appear in searches for `Pintos`. Thanks for considering this request.

**Comment by cpt-chewieeee** [comment]
Thank Prof. Ousterhout for reaching out, 

I will not take this repo nor will I make this repo private because its one of my project that reflects my area of concentration in OS. However, I did change the name of this repo to "POS_project". 

I dont know if you already knew about this, but there's a github user that goes by 'rtwilson' who has all four parts of the Pintos solution on his account. To my knowledge, rtwilson's Pintos is the only version that is 100% completed with all tests passing. rtwilson was the only page that appear when I did a code.google.com search on Pintos a few years back. The reason why I'm mentioning this is because rtwilson is affiliated with Stanford. 

Thanks.
